### DISCUSSION TITLE
suricata issues

### DISCUSSION THREAD

**Discussion by ktroberson** [Security-Onion-Solutions/securityonion#4870] [Unsupported Versions]
I am not receiving logs from suricata into the kibana dashboards.

**Comment by gcsrellis** [comment]
We need more information.  What kind of set up do you have?  Manager/Search node and Forward node?  Manager and separate search node and forward node?  Are you getting any traffic in from your sniffing interface?

**Comment by ktroberson** [comment]
Distributed setup one sensor, one management, two search nodes. We are getting traffic from zeek. Filebeats haven’t been modified. 

Keith Roberson
678-410-8830
